gis/grass7/Portfile

```diff
@@ -240,6 +237,13 @@ variant openblas description {Use OpenBLAS for BLAS/LAPACK} {
     configure.args-append   --with-liblapack=${prefix}/lib/libopenblas.dylib
 }
 
+variant zstd description {Add ZStandard compression support} {
+    depends_lib-append       port:zlib
+    configure.args-append    --with-zstd
+    configure.args-append    --with-zstd-includes=${prefix}/include
+    configure.args-append    --with-zstd-libs=${prefix}/lib
+}
+
```

Comment:
This adds minimal new dependencies. Just always add zlib support and remove the variant.

Reply:
Or is there some licensing implication?

Reply:
It's just a bit of caution for the user. With GRASS 7.8, ZSTD was made the default compression method for new raster maps **if** available (see https://grass.osgeo.org/grass78/manuals/r.compress.html). Making it a variant, a user deliberately chooses and should know the consequences of this.
As this option has not been available since introduction of 7.8.0, I believe it'd better be a variant till next major version bump.